pkg/releaseutil/manifest.go

```diff
@@ -31,17 +31,19 @@ type SimpleHead struct {
 	} `json:"metadata,omitempty"`
 }
 
+var sep = regexp.MustCompile("(?:^|\\s*\n)---\\s*")
+
 // SplitManifests takes a string of manifest and returns a map contains individual manifests
 func SplitManifests(bigfile string) map[string]string {
 	// This is not the best way of doing things, but it's how k8s itself does it.
```

Comment:
I am guessing that since you changed this, it no longer holds true. Can you update this comment to reflect the new code?